src/compas_rrc/ghpython/components/CompasRrc_MoveToFrame/metadata.json

```diff
@@ -0,0 +1,41 @@
+
+{
+    "name": "Move To Frame",
+    "nickname": "Move",
+    "category": "COMPAS RRC",
+    "subcategory": "Motion instructions",
+    "description": "Move robot to the specified frame in cartesian space.",
+    "exposure": 2,
+
+    "ghpython": {
+        "isAdvancedMode": true,
+        "iconDisplay": 2,
+        "inputParameters": [
+            {
+                "name": "frame",
+                "description": "Target frame."
+            },
+            {
+                "name": "speed",
+                "description": "Integer specifying TCP translational speed in mm/s.",
+                "typeHintID": "int"
```

Comment:
not float?

Reply:
not float... ABB RAPID is an inflexible friend sometimes... :P

Reply:
but 
```
if not speed or speed < 0.01:
            add_warning('Speed must be greater or equal to 0.01 mm/s.')
```